themes-default/slim/test/helpers/setup.js

```diff
@@ -6,6 +6,11 @@ const hooks = require('require-extension-hooks');
 // Setup browser environment
 browserEnv();
 
+// Setup document variables
+const baseElement = document.createElement('base');
+baseElement.setAttribute('href', 'http://localhost:8081');
+document.head.appendChild(baseElement);
```

Comment:
How are we going to test a `webRoot` since the base will be setup before anything else?

Reply:
`AppLink doesn't use `web-root`,
`Asset` will, so in that case we should be able to just add:
```js
document.body.setAttribute('web-root', '/medusa');
document.body.setAttribute('api-key', 'abcdef123456');
```